Package.swift

```diff
@@ -2,7 +2,7 @@
 import PackageDescription
 
 let package = Package(
-    name: "Promises",
+    name: "Promise",
```

Comment:
what effect does changing this package name have? i was trying to make sure the package name was the same as the cocoapods name.

Reply:
Changes to package name in Swift Package Manager changes bugger all.

Reply:
yea it didnt fix the issue. wasn't sure why but It seems you debunked the linux testing issue! 👍